src/test/java/org/wso2/lsp4intellij/utils/UtilsTest.java

```diff
@@ -0,0 +1,55 @@
+package org.wso2.lsp4intellij.utils;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.ArrayList;
+
+public class UtilsTest {
+
+  @Test
+  public void testConcatenateArrays() {
+    Assert.assertArrayEquals(new Object[]{new String[]{"a", "b"},
+            new String[]{"c", "d"}}, new Utils().concatenateArrays(
+                    new String[]{"a", "b"}, new String[]{"c", "d"}));
+  }
+
+  @Test
+  public void testParseArgsSingleQuote() {
+    Assert.assertArrayEquals(new String[] {"\'\'"}, Utils.parseArgs(
+            new String[]{"\'\'"}));
```

Comment:
Personal preference but I normally start a new line after the comma in the Assert.
`Assert.assertArrayEquals(new String[] {"\'\'"},`
`                Utils.parseArgs(new String[]{"\'\'"}));`